cpp/src/binaryop/binaryop.cpp

```diff
@@ -20,6 +20,7 @@
 #include "compiled/binary_ops.hpp"
 #include "jit/code/code.h"
 #include "jit/util.hpp"
+#include "thrust/optional.h"
```

Comment:
Move this down to after the <rmm> includes, and before <string>, and use angle brackets.

Reply:
Done.